petab/observables.py

```diff
@@ -1,8 +1,10 @@
 """Functions for working with the PEtab observables table"""
 
-from typing import Union
+from typing import Union, Set
 
+import libsbml
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'libsbml'](https://app.codacy.com/manual/dweindl/PEtab/pullRequest?prid=4911750)